src/components/YieldPools/ProMMFarmGroup.tsx

```diff
@@ -330,15 +332,15 @@ const Row = ({
               {token0?.symbol} - {token1?.symbol}
             </Text>
 
-            {farm.startTime > currentTimestamp && (
+            {/* farm.startTime > currentTimestamp && (
               <MouseoverTooltip
                 text={'Starting In ' + getFormattedTimeFromSecond(farm.startTime - currentTimestamp)}
                 width="fit-content"
                 placement="top"
               >
                 <Clock size={14} style={{ marginLeft: '6px' }} />
               </MouseoverTooltip>
-            )}
+             ) */}
```

Comment:
why not remove this

Reply:
business requirement. bring back ending in column so this is temporary hide